### PR TITLE
Updated tests

### PR DESCRIPTION
**Pull request by lee-delarm6**
Updated tests

## Summary

- Improved test targetting and updated text check for new section
- This only affects tests

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/64962

## Testing done

- Updated test data
- Ran and verified new tests run properly

## Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).

### CODE REVIEW

File: src/applications/check-in/tests/e2e/pages/NextOfKin.js [hunk 1]
```diff
@@ -49,16 +49,22 @@ class NextOfKin {
...
+        );
     },
     preCheckIn: () => {
-      cy.get('div[data-testid="additional-info"]').should(
-        'contain.text',
-        'If this isn’t your correct information, select No and a staff member can help you update your information on the day of your appointment.',
-      );
+      this.openAdditionalInfo();
+      cy.get('div[data-testid="additional-info"]')
+        .should('be.visible')
```
Comment (wullaski) [inline src/applications/check-in/tests/e2e/pages/NextOfKin.js:63]: For the `be.visible` asserts we should check the visibility on a div or something within it that is visible after you open the drawer.

**Review by wullaski — CHANGES_REQUESTED** [review]
There's an opportunity here to add another file that tests the next of kin display on day of and uses the dayOf method to test for the correct text. We could also test the other things that the `src/applications/check-in/pre-check-in/tests/e2e/next-of-kin-display/next.of.kin.display.cypress.spec.js` file tests on the day-of app.

File: src/applications/check-in/tests/e2e/pages/NextOfKin.js (hunk 1 above)
Comment (lee-delarm6) [reply src/applications/check-in/tests/e2e/pages/NextOfKin.js:63]: True, gotcha

**Comment by lee-delarm6** [comment]
All items addressed now 👍

**Review by rsmithadhoc — APPROVED** [review]

**Review by wullaski — APPROVED** [review]
LGTM!

**Pull request merged**